src/api/companies/get.js

```diff
@@ -0,0 +1,14 @@
+import { collections } from '../../firebase';
+
+export default function (user_uid) {
+  return new Promise((resolve, reject) => {
+    collections.companies
+      .where("user_id", "==", user_uid.toString())
+      .get()
+      .then((queryResult) => {
+        resolve(queryResult); // data()
+      }).catch((error) => {
+        reject(error);
+      })
+  });
+}
```

Comment:
Hey man, what do you think of returning just the result from `get`? It is a Promise by default, and we are not transforming anything. Not on the response, nor on an error.

What do you think of having the code like this:

```suggestion
export default function (user_uid) {
  return collections.companies
      .where("user_id", "==", user_uid.toString())
      .get();
}
```

Reply:
As I changed the companies logic's main structure, I entirely removed this function because we are not using `companies` collection anymore.